scripts/playwright_auth.sh

```
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Opens a headed browser for SSO login so that authentication cookies are
+# saved to the shared Playwright profile.  Run this when the headless
+# screenshot flow detects that the user is not authenticated.
+
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+source "$SCRIPT_DIR/playwright_common.sh"
+
+export CHROME_PATH="${CHROME_PATH:-$(detect_chrome)}"
+export PROFILE_DIR="$(cd "$SCRIPT_DIR/.." && pwd)/.playwright-mcp/profile"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)